wsgi.py

```diff
@@ -1,5 +1,6 @@
 """WSGI Server file."""
-from c3po.manage import app
+from c3po.manage import app, sched
 
 if __name__ == "__main__":
     app.run()
+    sched.start()
```

Comment:
How do we start this? Does it start along with `flask run` or is it separate?

Reply:
Not a big deal. Doesn't need to change. Just understanding.

Reply:
Yes, `flask run` starts the scheduler as well along with the flask application. Running `python c3po/manage.py` will create an instance of the scheduler but doesn't start it. It waits for flask run to start the scheduler and run jobs.

Reply:
Nope, I need to run the function `sched.start()` while creating the app instance. Otherwise the scheduler is created but isn't started. Let me change this.